docs/learn/distributed-authority-webgl.md

```diff
@@ -31,6 +31,10 @@ You also need the following packages and minimum versions for a WebGL distribute
 
 Follow the instructions on the [distributed authority general quickstart](distributed-authority-quick-start.md) page.
 
+## Configure the transport to use websockets
+
+Enable the "Use Web Sockets" checkbox on the **DistributedAuthorityTransport** component that was added to the *NetworkManager* object. ![enable web sockets](/img/learn/distributed-authority-quick-start/use-websockets.png)
```

Comment:
```suggestion
Enable the **Use Web Sockets** checkbox on the **DistributedAuthorityTransport** component that was added to the *NetworkManager* object. ![enable web sockets](/img/learn/distributed-authority-quick-start/use-websockets.png)
```
UI elements/names should be in bold formatting

Reply:
Thanks! I copied the formatting out of other places in the DA quickstart so I've gone through and updated all the formatting in that file so I don't confuse myself in the future 😄

Reply:
Consistency, in this economy?! (thanks tho; I've been meaning to do a pass for it throughout the whole docs, but it's just never been a high enough priority)

Reply:
Gotta save future Emma